sway/tree/container.c

```diff
@@ -1225,12 +1225,18 @@ void container_discover_outputs(struct sway_container *con) {
 }
 
 enum sway_container_layout container_parent_layout(struct sway_container *con) {
+	if (container_is_floating(con)) {
```

Comment:
I'll do a more thorough review of this tomorrow to ensure it won't break anything, before I approve it. 

However if it is valid, this can be simplified to:

```c
enum sway_container_layout container_parent_layout(struct sway_container *con) {
    return con->parent ? con->parent->layout : L_NONE;
}
```

Reply:
Sounds good. Though, I'm not sure if that simplification is correct: a toplevel container will have a null `con->parent`, but it is a child of the workspace, so should return `con->workspace->layout`.

For example, in a workspace repr of `T[A]`, I'd expect `container_parent_layout(A) == L_TABBED`.

Reply:
Oh yup. I guess I should get some sleep before I try reviewing anything else...

Reply:
Gentle ping :)